wrs/ceph-radosgw.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
```

Comment:
Now seems like a good time to change this directory name to something that isn't wrs